packages/nomad/package.yaml

```
@@ -0,0 +1,42 @@
+---
+name: nomad
+description: |
+  Nomad is an easy-to-use, flexible, and performant workload orchestrator that can deploy a mix of microservice,
+  batch, containerized, and non-containerized applications. Nomad is easy to operate and scale and has native Consul
+  and Vault integrations.
+homepage: https://www.nomadproject.io/
+licenses:
+  - BUSL-1.1
+languages:
+  - Nomad
+categories:
+  - Formatter
+  - Runtime
```

```suggestion
  - Linter
  - Runtime
```

`nomad job validate` actually does syntax checking and can be used for linting